providers/azure/helpers.go

```diff
@@ -20,10 +20,16 @@ import (
 const DirDelim = "/"
 
 func getContainerClient(conf Config) (*container.Client, error) {
-	dt, err := exthttp.DefaultTransport(conf.HTTPConfig)
+	// Check if a roundtripper has been set in the config
+	// otherwise build the default transport.
```

Comment:
I feel like this is discernable from the code itself so we can probably skip comments.
```suggestion
```